sklearn/utils/_array_api.py

```diff
@@ -1055,3 +1068,18 @@ def _modify_in_place_if_numpy(xp, func, *args, out=None, **kwargs):
     else:
         out = func(*args, **kwargs)
     return out
+
+
+def _bincount(xp, array, weights=None, minlength=None):
```

Comment:
I think `xp` should be the last param like many other methods here.

Reply:
We could add it as the last parameter but it has to made optional then. Currently the way this function is implemented it is kind of required. Otherwise we would need to extract xp from the array once again in case it is None.